sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -80,6 +81,28 @@ pi_result cuda_piPluginGetLastError(char **message) {
   return ErrorMessageCode;
 }
 
+// Optimization strings.
+static const char *EmptyStr = "";
+
+// Returns plugin specific backend option.
+// Current support is only for optimization options.
+// Return empty string for cuda.
+// TODO: Determine correct string to be passed.
+pi_result cuda_piPluginGetBackendOption(pi_platform platform,
+                                        const char *frontend_option,
+                                        const char **backend_option) {
+  using namespace std::literals;
+  if (frontend_option == nullptr)
+    return PI_ERROR_INVALID_VALUE;
+  if (frontend_option == "-O0"sv || frontend_option == "-O1"sv ||
+      frontend_option == "-O2"sv || frontend_option == "-O3"sv ||
+      frontend_option == ""sv)) {
+    *backend_option = EmptyStr;
```

Comment:
is there really no opt-level options in CUDA?

Reply:
I have not taken a good look at CUDA. Main focus of this change is level_zero and opencl.

Thanks